lumicks/pylake/calibration.py

```diff
@@ -46,7 +54,7 @@ def _sensor_type(self):
     def kind(self):
         kind = self.data.get("Kind", "Unknown")
         if kind == "Full calibration":
-            return "Active calibration" if self.active_calibration else "Passive calibration"
+            return "Active" if self.active_calibration else "Passive"
         else:
             return kind
```

Comment:
I noticed we have a mismatch between this type and `CalibrationResults`. I think the shorter version here is good; but `CalibrationResults.kind` returns `"active calibration" | "passive calibration"`. This is the kind of stuff that's gonna bite us because we have to maintain two different classes.

For this one in particular, I wonder if it's better to move it to the mixin and just test the type there. Not super great as far as coding practice, but at least it avoids this kind of mistake

Reply:
Yeah, let's move this one to the mixin.